.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   rules: {
     indent: ['error', 2, { SwitchCase: 1 }],
     'linebreak-style': ['error', 'unix'],
+    'max-len': ['error', { code: 100, ignoreComments: true }],
```

Comment:
how do you feel about 120?

Reply:
120 is good, I'll make the change in my next branch